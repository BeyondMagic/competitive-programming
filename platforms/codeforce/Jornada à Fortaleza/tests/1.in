2
5 5
1 2 1
1 5 2
2 3 2
3 4 2
4 5 6
7 8
1 2 1
2 5 2
5 7 1
1 3 3
3 6 3
6 7 3
2 6 2
3 5 2
