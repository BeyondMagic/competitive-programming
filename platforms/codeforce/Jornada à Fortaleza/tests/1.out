13 5 2
1 2 3 4 5 
1 5 
12 4 4
1 2 6 7 
1 3 5 7 
